content/docs/next-release-v2/cassandra.md

```diff
@@ -14,24 +14,8 @@ Cassandra also has the following officially supported resources available from t
 - [Kubernetes Operator](https://github.com/k8ssandra/cass-operator) from DataStax
 
 #### Configuration
```

Comment:
this is the first header in the file, shouldn't it be L1 (`#`)?

Reply:
Sorry I don't understand the comment. Do you want a title for the file called Cassandra too?

Reply:
`####` is level-4 header, a page should start with level-1 header `#`